src/renderer/components/inputs/InputContainer.tsx

```diff
@@ -108,12 +108,28 @@ export const InputContainer = memo(
 
         const parentTypeColor = useMemo(() => {
             if (connectedEdge) {
-                const parentNode: Node<NodeData> = getNode(connectedEdge.source)! as Node<NodeData>;
+                const parentNode: Node<NodeData> | undefined = getNode(connectedEdge.source);
                 const parentOutputId = parseSourceHandle(connectedEdge.sourceHandle!).inOutId;
-                const parentType = functionDefinitions
-                    .get(parentNode.data.schemaId)!
-                    .outputDefaults.get(parentOutputId)!;
-                return getTypeAccentColors(parentType, typeDefinitions, isDarkMode)[0];
+                if (parentNode) {
+                    const parentDef = functionDefinitions.get(parentNode.data.schemaId);
+                    if (!parentDef) {
+                        return getTypeAccentColors(
+                            NeverType.instance,
+                            typeDefinitions,
+                            isDarkMode
+                        )[0];
```

Comment:
This seems like a lot to get the default color. Maybe we could just export the default color and use it directly?